lib/fireblocks/api/token.rb

```diff
@@ -56,7 +60,31 @@ def nonce
     end
 
     def rsa_private
-      OpenSSL::PKey::RSA.new(Fireblocks.configuration.private_key)
+      OpenSSL::PKey::RSA.new(private_key)
```

Comment:
You are probably correct, I am not finding them used. However, I think I will leave it alone because we have been having some issues with fireblocks keys so I don't want to make any changes that are not absolutely necessary in order to get this ticket working until that is all sorted out. Maybe it could be a tech debt ticket.

Reply:
Totally fair - I think it's bordering `nit` anyways since it technically wouldn't hurt anything by not being there. 

Out of curiosity though, since I know Grant was working on some way of testing Fireblocks with Jan (if I remember correctly?) is there a way of testing this in a tech debt ticket? Semaphore probably wouldn't show any errors if Fireblocks would have them.. afaik